index.js

```diff
@@ -202,15 +202,16 @@ const pkg = getPackageJson();
     console.log('newVersion 2:', newVersion);
     newVersion = `${tagPrefix}${newVersion}`;
     console.log(`newVersion after merging tagPrefix+newVersion: ${newVersion}`);
-    console.log(`::set-output name=newTag::${newVersion}`);
+    // Using sh as command instead of directly echo to be able to use file redirection
+    await runInWorkspace('sh', ['-c', `echo "newTag=${newVersion}" >> $GITHUB_OUTPUT`]);
     try {
       // to support "actions/checkout@v1"
       if (process.env['INPUT_SKIP-COMMIT'] !== 'true') {
         await runInWorkspace('git', ['commit', '-a', '-m', commitMessage.replace(/{{version}}/g, newVersion)]);
       }
     } catch (e) {
       console.warn(
-        'git commit failed because you are using "actions/checkout@v2"; ' +
+        'git commit failed because you are using "actions/checkout@v2" or later; ' +
```

Comment:
I have added this for the sake of clarity. I was using `actions/checkout@v3` and the message was a bit misleading.